astroid/brain/brain_numpy_ma.py

```diff
@@ -21,6 +21,9 @@ def numpy_ma_transform():
     import numpy.ma
     def masked_where(condition, a, copy=True):
         return numpy.ma.masked_array(a, mask=[])
+
+    def masked_invalid(a, copy=True):
```

Comment:
I know it is still work in progress but I noticed the docstring above is now out of date since it specifically mentions `masked_where`.